cypress/support/function.js

```diff
@@ -5,8 +5,23 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 import crypto from "crypto";
-const gridcapaUser = Cypress.env('GRIDCAPA_USER');
-const gridcapaPassword = Cypress.env('GRIDCAPA_PASSWORD');
+
```

Comment:
maybe we can think of a new suitable name for this class, kind of processes-util.js?